gm4_area_markers/pack.mcmeta

```diff
@@ -0,0 +1,24 @@
+{
+  "pack":{
+    "pack_format":0,
+    "description":"A Gamemode 4 Module"
+  },
+  "module_name":"Area Markers",
+  "module_id":"area_markers",
+  "site_description":"Set up floating messages with a simple name tag!",
```

Comment:
The site description should be changed to a different message than the Holographic Tags description